modules/release-notes/partials/docs-server-7.2.6-release-note.adoc

```diff
@@ -0,0 +1,164 @@
+[#release-726]
+== Release 7.2.6 (September 2024)
+
+Couchbase Server 7.2.6 was released in September 2024.
+This maintenance release contains the following fixes.
+
+For details on new features and enhancements, see the  xref:introduction:whats-new.adoc[What's New page].
+
+
+=== Fixed Issues
+
+==== Cluster Manager
+[#table-fixed-issues-726-cluster-manager, cols="10,40,40"]
+|===
+|Issue | Description | Resolution
+
+| https://issues.couchbase.com/browse/MB-62219[MB-62219]
```

Comment:
Change link to: https://jira.issues.couchbase.com/browse/MB-62219

Also, we need to clean up this description...I see that Timofey suggested: Fixed inability to rotate root certificate when new root uses the same private key as the old root.

Reply:
Maybe for the problem description, add:

You could not rotate root certificates when a new root used the same private key as the old root.

THen in the resolution column: THis issue has been resolved.